### PR TITLE
Add mise version manager config

### PR DESCRIPTION
**Pull request by wendevlin**
## Proposed change
Mise is a version manager to use for multiple languages and tools, for example nodejs, python, go,...
The benefit in the home assistant frontend is that mise can install the correct node version and the correct yarn version at once. This is defined in the `.mise.toml` and can be done with a simple `mise install`

See mise documentation [here](https://mise.jdx.dev/getting-started.html)

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Additional information

- Link to documentation pull request: https://github.com/home-assistant/developers.home-assistant/pull/2341

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: .mise.toml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[tools]
+node = "lts-iron"
```
Comment (bramkragten) [inline .mise.toml:2]: mise will use `.nvmrc` if not present, so I think we can just remove this and keep the version in one place?

File: .mise.toml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+yarn = "4"
```
Comment (bramkragten) [inline .mise.toml:3]: yarn should no longer be (globally) installed, and we should use corepack, the version of yarn is defined in `package.json` `packageManager`. Can we use that instead?

Comment (wendevlin) [reply .mise.toml:3]: good point. Then we can get rid of .mise.toml, corepack is there without the setting: MISE_NODE_COREPACK

**Comment by wendevlin** [comment]
mise fallbacks to .nvmrc and yarn can be used via corepack, so .mise.toml isn't needed

**Pull request closed without merging**